src/domain/entities/noteHistory.ts

```diff
@@ -1,6 +1,18 @@
 import type { NoteInternalId, Note, NotePublicId } from './note.js';
 import type User from './user.js';
 
+interface UserMeta {
```

Comment:
I think, id can be useful as well

Reply:
it is outside of the user meta,  but included in history meta objects

Reply:
maybe remove `id` from outside and put to "user"?

Reply:
that would really complicate two types relations (noteHistory record and noteHistoryMeta), since we already store userId as a separate field, i guess that would be just unneeded